apps/crn-server/src/data-providers/email-notification-service.ts

```diff
@@ -271,7 +289,10 @@ export class EmailNotificationService {
       ].filter(Boolean) as string[];
 
       let openScienceRecipients = isProduction
-        ? ['openscience@parkinsonsroadmap.org']
+        ? ([
+            openScienceDL,
+            ...(isDiscussionCreatedAction ? assignedOSMembersEmails : []),
```

Comment:
In the ticket it says `Grantee starts a discussion` but `isDiscussionCreatedAction` can be `discussion_created_by_os_member` also, right?
<img width="851" height="190" alt="Image" src="https://github.com/user-attachments/assets/c009328c-994f-4b1f-bd0c-76e7074551ed" />